docs/markdown/Reference-manual.md

```diff
@@ -1245,6 +1245,12 @@ Keyword arguments are the following:
 - `workdir` absolute path that will be used as the working directory
   for the test
 
+- 'suite' label attached to this test. `meson test --suite suite_name`
+  run all the tests with suite defined as `suite_name`. Note that by
+  default, the full suite name is `project_name:suite_name`, so if you
+  want to select the test suite of a given (sub)project only, you have
+  to run 'meson test --test project_name:suite_name'
```

Comment:
This part possibly belongs more properly in the discussion of the `meson test` command in Unit-tests.md

A clearer phrasing might be to say that the suite name is qualified by a (sub)project name, (and that (sub)project name can be omitted if it is the top-level project, with 0.46 and later)